config/initializers/devise.rb

```diff
@@ -0,0 +1,313 @@
+# frozen_string_literal: true
```

Comment:
- [ ] While trying to create the DB and start the project, I got this error as seen in the screenshot below. The error points at an `uninitialized constant Devise`. I'll suggest you fix this as it prevents your project from running. :+1: :100: :+1: